### DISCUSSION TITLE
Feature Request: Add support for post quantum encryption

### DISCUSSION THREAD

**Discussion by CitizenRacer** [homeassistant-apps/app-cloudflared#344] [Ideas]
CloudFlare released this blog post today: https://blog.cloudflare.com/post-quantum-crypto-should-be-free/

Turning on PQ safe crypto looks pretty easy:

> If you use [Tunnels with cloudflared then upgrading to PQC is super simple](https://blog.cloudflare.com/post-quantum-tunnel/). Make sure you’re on at least version [2022.9.1](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/) and simply run cloudflared --post-quantum.

**Comment by brenner-tobias** [comment]
Great suggestion, thanks a lot. I will have a look and see if there are any downsides (e.g. performance), since it is not really necessary right now if we are being honest ;-). Nevertheless, if there are none, I think we should implemented it as default.
